administrator/language/en-GB/en-GB.mod_quickicon.ini

```diff
@@ -8,33 +8,50 @@ MOD_QUICKICON_ADD_NEW="New %s"
 MOD_QUICKICON_AMOUNT_THSD="THSD"
 MOD_QUICKICON_ARTICLE_MANAGER="Articles"
 MOD_QUICKICON_ARTICLE_MANAGER_1="Article"
-MOD_QUICKICON_CACHE="Cache (MB)"
+MOD_QUICKICON_ARTICLE_SRONLY="Articles: %s articles are available"
+MOD_QUICKICON_ARTICLE_SRONLY_0="Articles: No  is available"
+MOD_QUICKICON_ARTICLE_SRONLY_1="Articles: One article is available"
+MOD_QUICKICON_CACHE="Cache"
+MOD_QUICKICON_CACHE_SRONLY="Cache: %s MB are used"
+MOD_QUICKICON_CACHE_SRONLY_1="Cache: %s MB is used"
+MOD_QUICKICON_CACHE_SRONLY_0="Cache: The Cache is empty"
 MOD_QUICKICON_CATEGORY_ADD="New Category"
 MOD_QUICKICON_CATEGORY_MANAGER="Article Categories"
 MOD_QUICKICON_CATEGORY_MANAGER_1="Article Category"
+MOD_QUICKICON_CATEGORY_SRONLY="Article categories: %s categories are available"
+MOD_QUICKICON_CATEGORY_SRONLY_0="Article categories: No category is available"
+MOD_QUICKICON_CATEGORY_SRONLY_1="Article categories: One category is available"
 MOD_QUICKICON_CHECKIN="Checkin"
 MOD_QUICKICON_CHECKINS="Global Checkin"
-MOD_QUICKICON_CHECKIN_ALL="Global Checkin"
+MOD_QUICKICON_CHECKIN_SRONLY="Global Checkin: %s Items are locked."
+MOD_QUICKICON_CHECKIN_SRONLY_0="Global Checkin: No Items are locked."
+MOD_QUICKICON_CHECKIN_SRONLY_1="Global Checkin: One Item is locked."
 MOD_QUICKICON_CONFIGURATION="Configuration"
 MOD_QUICKICON_CONTENT="Content"
-MOD_QUICKICON_EXTENSIONS="Extensions"
-MOD_QUICKICON_EXTENSION_MANAGER="Extension Manager"
 MOD_QUICKICON_GLOBAL_CONFIGURATION="Global Configuration"
 MOD_QUICKICON_GROUP_DESC="The group of this module (this value is compared with the group value used in <strong>Quick Icons</strong> plugins to inject icons). The 'mod_quickicon' group always displays the Joomla! core icons."
 MOD_QUICKICON_GROUP_LABEL="Group"
 MOD_QUICKICON_HEADER_ICON_LABEL="Header Icon"
 MOD_QUICKICON_INSTALL_EXTENSIONS="Install Extensions"
-MOD_QUICKICON_LANGUAGE_MANAGER="Languages"
+MOD_QUICKICON_LOAD_PLUGINS_LABEL="Load 3rd party plugins"
```

Comment:
plugins? do you mean extensions? Is this even used?

Reply:
We don't have icons for languages and extensions. So better delete the language keys